adapters/LiftoffMonetize/Public/Headers/VungleAdNetworkExtras.h

```diff
@@ -26,9 +26,10 @@
 @property(nonatomic, copy) NSString *_Nullable playingPlacement;
 
 /*!
- * @brief NSInteger that will be passed to alter the privacy icon position for native ads.
- * @discussion Optional. topLeft = 1, topRight = 2, bottomLeft = 3, bottomRight = 4
+ * @brief GAAdChoicesPosition enum that will be passed to alter the privacy icon position for
```

Comment:
@crbinsf Sorry for the late message, a tiny typo: `GAAdChoicesPosition` -> `GADAdChoicesPosition`. Thanks a lot. : )